packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -70,41 +76,46 @@ The expandable list adds an expansion panel to every row to reveal more details
 Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
 
 ## Tables
-* **Standard table:** Combines the styling of a list view with behaviors that are commonly associated with a table (sometimes referred to as a hybrid list)
-* **Compact table:** Maximizes the amount of data that can be displayed in a small space
 
-### Standard table
+### How to use
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
-![standard table](./img/standard-data-table.png)
+### Table capabilities
+Every [compact](#when-to-use-compact-vs-basic-spacing) or [basic](#when-to-use-compact-vs-basic-spacing) table can be extended with these functionalities. 
+
+* **[Expandable data table](#expandable-data-table)**
```

Comment:
There is an issue with this link (I think it's because the actual heading for that section is "Expandable table" and not "Expandable **data** table"!) So should be #expandable-table !